po/tr.po

```diff
@@ -2,20 +2,21 @@
 # Copyright (C) 2020
 # This file is distributed under the same license as the drawing package.
 # Serdar Sağlam <teknomobil@yandex.com>, 2018, 2019, 2020.
+# Serkan ÖNDER <serkanonder@outlook.com>, 2021
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: drawing\n"
+"Project-Id-Version: drawing 0.6.5\n"
```

Comment:
i would prefer that you don't put a version number here: this file will continue to exist even after the release of 0.6.5

actually the code from which you've translated isn't the version 0.6.5, but the unstable 0.7.0 version